src/Action.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const Log = props => {
+  // Setup is similar to Contact.js
+  return (
+    <li className="Log">
+      <div className="lg-col-3">
+        <button
+          // Button to delete action
+          onClick={() => props.onDeleteAction(props.log)}
```

Comment:
This is looking good, but I don't think you passed a prop named `onDeleteAction`, looks like you pass it as `onClick` from your `ActionLog` component